tubesync/sync/models.py

```diff
@@ -422,7 +422,7 @@ class IndexSchedule(models.IntegerChoices):
         help_text=_('List of subtitles langs to download, comma-separated. Example: en,fr or all,-fr,-live_chat'),
         validators=[
             RegexValidator(
-                regex=r"^(\-?[\_\.a-zA-Z]+,)*(\-?[\_\.a-zA-Z]+){1}$",
```

Comment:
@meeb While looking into this, I was left wondering why it's using `A*A{1}` as a construction instead of simply `A+` as was used for the internal characters.

Reply:
This was from a previous PR, I'm fine with this being corrected. Thanks!

Reply:
Alright. The proposed change is in #574.